src/egon/data/datasets/plotdatascn.py

```diff
@@ -0,0 +1,196 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+# File description
+"""
+Created on Tue May 24 14:42:05 2022
+Plotdatascn.py defines functions to plot to provide a better context of the different parameters part of
+scenarios eGon2035 and eGon100RE .
+@author: Alonso
+
+"""
+
+
+    
+   
+   
+   
+import logging
+import os
+from matplotlib import pyplot as plt
+import matplotlib as mpl
+import pandas as pd
+from math import sqrt, log10
+from pyproj import Proj, transform
+from egon.data import db
+from egon.data.datasets import Dataset
+import egon.data.config
+import geopandas as gpd
+import tilemapbase
```

Comment:
Tilemapbase is not part of the standard libraries used in eGon-data. To include it you should modify setup.py. But before, please try to replace it with functions in the libraries we normally use.

Reply:
The problem is that we are using Tilemapbase to create an Osm map in the background of the plot. I took the idea from the file plot.py

I think it could be easier to modify setup.py...

Reply:
Setup.py does not includes Tilemapbase. 
Should I include it?

Reply:
Clara created plots with very similar characteristics of the ones you are trying to do. Please check with her how she managed to not use extra libraries. If not, we can just include Tilemapbase in setup.py. Additionally, there are some previous comments that need to be attended. Please let me know when everything is done to merge and close this issue.

Reply:
Carlos do I have to include Tilemapbase in setup.py from eGon-Data or eTraGo?

Reply:
I added Tilemapbase in setup.py from eGon-Data

Reply:
New libraries are supposed to be added to "install_requires" https://github.com/openego/eGon-data/blob/5e4a39915c7a45a3c51671be6754baee4e41443d/setup.py#L83. Additionally, there is a typo and you used capital letters. Since tilemapbase is not mandatory for running eGon-data, let's make it optional. Please delete the changes to setup.py and include a comment in plotdatascn.py clarifying the libraries that could be necessary additionally to generate the figures.

Reply:
ok, I already did it. Can you check it? Thanks Carlos